### PR TITLE
Improves language TGUI menu. Now it shows lang icon

### PR DESCRIPTION
**Pull request by EvilDragonfiend**
## About The Pull Request
Improves language TGUI menu.
Now it shows language icons!

and minor language menu code improvement

## Why It's Good For The Game
Better UX

## Testing Photographs and Procedure

image

image

## Changelog
:cl:
code: Language menu TGUI now shows language icon, and better readable description layout
code: Slightly improved language menu code
/:cl:

### CODE REVIEW

**Comment by Rukofamicom** [comment]
This is just rubbing salt in the wound now
😭

**Comment by itsmeow** [comment]
pls center the icons vertically with the text

**Comment by EvilDragonfiend** [comment]
> pls center the icons vertically with the text

I reuploaded the requested change. I can't make it better more because it uses `LabeledList` that isn't supposedly good for this table.

File: tgui/packages/tgui/interfaces/LanguageMenu.js [hunk 1]
```diff
@@ -67,25 +91,50 @@ export const LanguageMenu = (props, context) => {
...
+                            act('grant_language', {
+                              language_name: languageData.name,
+                            })
+                          }
+                        />
+                      }>
+                      <Box>{languageData.desc}</Box>
+                      <Box pl={4}>
+                        Chat key: ,{languageData.key} {!!each_language.shadow && '(Language gained from mob)'}
+                      </Box>
```
Comment (PowerfulBacon) [inline tgui/packages/tgui/interfaces/LanguageMenu.js]: Why do we need this here if we have it just above?

File: tgui/packages/tgui/interfaces/LanguageMenu.js [hunk 2]
```diff
@@ -67,25 +91,50 @@ export const LanguageMenu = (props, context) => {
...
+                      <Box pl={4} pb={1}>
```
Comment (PowerfulBacon) [inline tgui/packages/tgui/interfaces/LanguageMenu.js]: The intendation looks really weird here

**Review by PowerfulBacon — CHANGES_REQUESTED** [review]

File: tgui/packages/tgui/interfaces/LanguageMenu.js (hunk 1 above)
Comment (EvilDragonfiend) [reply tgui/packages/tgui/interfaces/LanguageMenu.js]: new players will not notice if there is a shortcut key for a language when they read a wall of texts.

Comment (EvilDragonfiend) [reply tgui/packages/tgui/interfaces/LanguageMenu.js]: oh wait. it's double.

**Comment by EvilDragonfiend** [comment]
image

**Review by PowerfulBacon — APPROVED** [review]

**Pull request merged**